config/default.template.json

```diff
@@ -35,5 +35,9 @@
     "bucket": "your GCS bucket name",
     "projectId": "your GCS project ID",
     "keyFile": "your credentials key file path"
+  },
+  "logger": {
+    "path": "logs",
+    "filename": "api-log.txt"
```

Comment:
Currently, it appends datetime in each record in the log file.

To separate daily log files, we may need [this package](https://github.com/winstonjs/winston-daily-rotate-file). Anyway, will do it as a separated PR.